pom.xml

```diff
@@ -68,6 +72,37 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Why did you add junit separately? That too twice?
Junit is already part of spring test.

Reply:
I deleted the repeated one.